README.md

```diff
@@ -1562,11 +1562,11 @@ require'lspconfig'.dartls.setup{}
     cmd = { "dart", "./snapshots/analysis_server.dart.snapshot", "--lsp" }
     filetypes = { "dart" }
     init_options = {
-      closingLabels = "true",
-      flutterOutline = "false",
-      onlyAnalyzeProjectsWithOpenFiles = "false",
-      outline = "true",
-      suggestFromUnimportedLibraries = "true"
+      closingLabels = false,
+      flutterOutline = false,
+      onlyAnalyzeProjectsWithOpenFiles = false,
+      outline = false,
+      suggestFromUnimportedLibraries = true
```

Comment:
@PatOConnor43 BTW, why some options were changed to false?

Reply:
I mentioned that briefly in the description. The additional features (closingLabels, flutterOutline, outline) require custom lsp callbacks that aren't supplied by the default config so I went ahead and turned them off.